modules/client-configuration/pages/package-management.adoc

```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
 . In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
     To compare with a stored profile, select the profile and click btn:[Compare].
 . To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.
+
+
+
+[[package-management-orphaned-packages]]
+== Orphaned Packages
+
+Orphaned packages are packages synchronized by {productname} that are not associated with a software channel.
+Therefore, orphaned packages are usually not available to {productname} clients and users cannot install such packages without additional effort.
+
+A package can become orphaned as a result of one of the following events:
+
+
+* A synchronized repository removes a package.
+  By default, {productname} 5.0 unlinks such package from the channel, but does not remove the package.
+  In {productname} 4.3, you can enable this behavior by using the [literal]``strict mode`` channel configuration.
```

Comment:
I would NOT keep this in the 5.0 version, it only confuses the user. Rather, have ONLY relevant instructions in the documents that are either 5.0 or 4,3.

Reply:
ok, done!